### PR TITLE
python310Packages.pillow: 9.4.0 -> 9.5.0

### PR DESCRIPTION
**Pull request by mweinelt**
https://github.com/python-pillow/Pillow/blob/9.5.0/CHANGES.rst

(Yes, I have seen 10.0.0, but 5 days old and breaking changes needs a python-updates cycle)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pillow/default.nix [hunk 1]
```diff
@@ -12,17 +13,25 @@
...
 
   src = fetchPypi {
     pname = "Pillow";
     inherit version;
-    hash = "sha256-ocLXeARI65P7zDeJvzkWqlcg2ULjeUX0BWaAMX8c0j4=";
+    hash = "sha256-v1SEedM2cm16Ds6252fhefveN4M65CeUYCYxoHDWMPE=";
   };
 
+  patches = [
+    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pillow/default.nix:28]: I guess this replaces https://github.com/NixOS/nixpkgs/pull/228493/files#diff-9c0332007fcf0d311ac6169461edeb3cc3f44e8bae1e891841a677d19c260981R30-R49 ?

**Review by SuperSandro2000 — APPROVED** [review]
@ofborg build python310Packages.pillow python311Packages.pillow

File: pkgs/development/python-modules/pillow/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/pillow/default.nix:28]: It does, as proposed in that PR.

**Comment by mweinelt** [comment]
> @ofborg build python310Packages.pillow python311Packages.pillow

That is very unrealistic on staging with stdenv rebuilds.

**Pull request merged**